File: sdkwork-app-sdk-typescript/src/types/collection-detail-vo.ts
import type { CollectionItemVO } from './collection-item-vo';

/** 合集详情响应 */
export interface CollectionDetailVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 合集ID */
  collectionId?: string;
  /** 合集名称 */
  name?: string;
  /** 合集描述 */
  description?: string;
  /** 合集类型 */
  type?: string;
  /** 封面图 */
  coverImage?: string;
  /** 是否公开 */
  isPublic?: boolean;
  /** 是否置顶 */
  isPinned?: boolean;
  /** 内容数量 */
  itemCount?: number;
  /** 浏览次数 */
  viewCount?: number;
  /** 收藏次数 */
  favoriteCount?: number;
  /** 父级ID */
  parentId?: number;
  /** 内容项列表 */
  items?: CollectionItemVO[];
}

File: sdkwork-app-sdk-go/types/collection_detail_vo.go
package types

// 合集详情响应
type CollectionDetailVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	CollectionId string `json:"collectionId"`
	Name string `json:"name"`
	Description string `json:"description"`
	Type string `json:"type"`
	CoverImage string `json:"coverImage"`
	IsPublic bool `json:"isPublic"`
	IsPinned bool `json:"isPinned"`
	ItemCount int `json:"itemCount"`
	ViewCount int `json:"viewCount"`
	FavoriteCount int `json:"favoriteCount"`
	ParentId int `json:"parentId"`
	Items []CollectionItemVO `json:"items"`
}
